app/components/works/authors_and_contributors_component.rb

```diff
@@ -0,0 +1,17 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Works
+  # A widget for managing both the authors (ordered) and unordered contributors to the work.
```

Comment:
Trivial tweak: can you move `(ordered)` in front of `authors`? Don't want to risk encouraging folks to infer that contributors are both unordered and ordered.

Reply:
done